hydra/test_utils/config_source_common_tests.py

```diff
@@ -10,6 +11,15 @@
 
 
 class ConfigSourceTestSuite:
+    def skip_overlap_config_path_name(self) -> bool:
+        """
+        Some config source plugins do not support config name and path overlap, for example:
+        /dateset.yaml
+        /dataset/cifar.yaml
+        is not allowed in configerator.
```

Comment:
do not mention configerator in the code.
just say that it's not supported by some config source plugins.

Reply:
removed. :)